src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -46,7 +46,7 @@
      * <code>
      * {@doc.settingsClassName}.Builder {@doc.settingsBuilderVarName} =
      *     {@doc.settingsClassName}.newBuilder();
-     * {@doc.settingsBuilderVarName}.{@doc.exampleApiMethodSettingsGetter}().getRetrySettingsBuilder()
+     * {@doc.settingsBuilderVarName}.{@doc.exampleApiMethodSettingsGetter}().getRetrySettings().toBuilder()
```

Comment:
Maybe call out in the commit message that there is a sample code fix (otherwise it's very buried)